components/serializer.rst

```diff
@@ -1242,6 +1242,31 @@ to ``true``::
     $result = $normalizer->normalize($dummy, 'json', [AbstractObjectNormalizer::SKIP_NULL_VALUES => true]);
     // ['bar' => 'notNull']
 
+Preventing ``null`` value fallback for nullable properties
```

Comment:
```suggestion
Preventing ``null`` Value Fallback for Nullable Properties
```

Reply:
@OskarStark Aren't all nouns written in lowercase in English (except countries and languages)? Or is this a convention of the Symfony documentation?

Reply:
For reference:
https://symfony.com/doc/current/contributing/documentation/standards.html#english-language-standards

Reply:
@OskarStark Thanks for the reference, I've committed the changes 🙂

Reply:
Thank you!